devDocs/developerGuide.t2t

```diff
@@ -195,8 +195,9 @@ The following few sections will talk separately about App Modules and Global Plu
 After this point, discussion is again more general.
 
 ++ Basics of an App Module ++
-App Module files have a .py extension, and are named the same as either the main executable of the application for which you wish them to be used or the package inside a host executable.
+App Module files have a .py extension, and in most cases should be named the same as either the main executable of the application for which you wish them to be used or the package inside a host executable.
 For example, an App Module for notepad would be called notepad.py, as notepad's main executable is called notepad.exe.
+If you want to use a single ap Module for a multiple executables, or the name of the executable conflicts with the standard Python import rules see the section on mapping an app Module to the executable below.
```

Comment:
Should remove the "a" before multiple.

Also "ap Module" should be "app Module". Actually, it should probably be "app module" or "App Module". I'm not sure it's a good idea to introduce yet a third casing for this phrase.

```suggestion
If you want to use a single app Module for multiple executables, or the name of the executable conflicts with the standard Python import rules see the section on mapping an app Module to the executable below.
```